docs/content/developers/release-management.md

```diff
@@ -168,7 +168,7 @@ docker run --rm -v $PWD:/tmp/chocolatey -w /tmp/chocolatey linuturk/mono-choco p
 
 The AUR repository normally updates with the release process, so nothing needs to be done.
 
-However, you can manually publish the release to [the DDEV AUR repository](https://aur.archlinux.org/packages/ddev-bin/). The README.md in the AUR Git repository (`ssh://aur@aur.archlinux.org/ddev-bin.git` or `https://aur.archlinux.org/ddev-bin.git`) has instructions on how to update, including how to do it with a Docker container, so it doesn’t have to be done on an ArchLinux or Manjaro VM.
+However, you can manually publish the release to [the DDEV AUR repository](https://aur.archlinux.org/packages/ddev-bin/). The README.md in the AUR Git repository (`ssh://aur@aur.archlinux.org:ddev-bin.git` or `https://aur.archlinux.org/ddev-bin.git`) has instructions on how to update, including how to do it with a Docker container, so it doesn’t have to be done on an ArchLinux or Manjaro VM.
```

Comment:
```suggestion
However, you can manually publish the release to [the DDEV AUR repository](https://aur.archlinux.org/packages/ddev-bin/). The README.md in the AUR Git repository (`ssh://aur@aur.archlinux.org/ddev-bin.git` or `https://aur.archlinux.org/ddev-bin.git`) has instructions on how to update, including how to do it with a Docker container, so it doesn’t have to be done on an ArchLinux or Manjaro VM.
```

I reverted it back because:
```
git clone ssh://aur@aur.archlinux.org/ddev-bin.git
Cloning into 'ddev-bin'...
...
```
And
```
git clone ssh://aur@aur.archlinux.org:ddev-bin.git
Cloning into 'ddev-bin'...
fatal: no path specified; see 'git help pull' for valid url syntax
```

Reply:
Thought this was fixed in master already. Thanks.

Reply:
It's good in master, it was changed here in this PR.
`git clone ssh://aur@aur.archlinux.org:ddev-bin.git` is not going to work.

Reply:
Oh I see, you have a different approach. I think the ssh:// just needs to be removed? Not sure why it's not actually just a link to the README. I would think this:

```suggestion
However, you can manually publish the release to [the DDEV AUR repository](https://aur.archlinux.org/packages/ddev-bin/). The README.md in the AUR Git repository (`https://aur.archlinux.org/ddev-bin.git`) has instructions on how to update, including how to do it with a Docker container, so it doesn’t have to be done on an ArchLinux or Manjaro VM.
```

Reply:
Yes, we can remove it.